app/logic/adminCreateEvent.py

```diff
@@ -0,0 +1,23 @@
+from app.models import*
+from app.models.term import Term
+from app.models.user import User
+
+
+def getTermDescription():
+
+    termDescriptions = Term.select(Term.description)
+    descriptions = [des.description for des in termDescriptions.objects()]
+
+    return descriptions
+
+def getCurrentTerm():
+
+    currentTerm = Term.select(Term.description).where(Term.isCurrentTerm)
+    term = [t.description for t in currentTerm.objects()] #This could be done better. but how??
+    return term[0]

```

Comment:
`currentTerm = Term.get(isCurrentTerm=True)`
`return currentTerm.description`

I think you should return the whole term object though, rather than just the description. In which case the function is a one-liner. ` return Term.get(isCurrentTerm = True)`